tests/store/artifact/test_dbfs_rest_artifact_repo.py

```diff
@@ -81,6 +81,24 @@ def test_init_validation_and_cleaning(self):
             with pytest.raises(MlflowException):
                 DbfsRestArtifactRepository('s3://test')
 
+    def test_init_get_host_creds_with_databricks_profile_uri(self):
```

Comment:
need to add something like this to v2

Reply:
and also for `runs_artifact_repo`, `databricks_artifact_repo`, `models_artifact_repo`